src/pymordemos/coercivity_estimation_scm.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+"""Simplified version of the thermalblock demo to showcase the successive constraints method.
+
+Usage:
+  coercivity_estimation_scm.py ALG SNAPSHOTS RBSIZE TEST
+"""
```

Comment:
The usage part should be auto-generated by typer. Also, `main` does not seem to accept an algorithm. Did you commit everything?

Reply:
That was a leftover from a previous version.